src/views/dialogs/DepositDialog2/DepositSteps.tsx

```diff
@@ -0,0 +1,77 @@
+import { ReactNode } from 'react';
+
+import ConnectingLine from '@/icons/connecting-line.svg';
+
+import { Icon, IconName } from '@/components/Icon';
+import { LoadingSpinner } from '@/components/Loading/LoadingSpinner';
+
+import { DepositStep } from './utils';
+
+type DepositStepsProps = {
+  steps: DepositStep[];
+  currentStep: number;
+  showRetry: boolean;
+  onRetry: () => void;
+};
+
+const STEP_TYPE_TO_INFO: { [type: string]: { title: string; icon: ReactNode } } = {
+  network: {
+    title: 'Switch networks',
+    icon: <Icon size="1.25rem" iconName={IconName.Switch} />,
+  },
+  approve: {
+    title: 'Approve USDC',
+    icon: <Icon size="1.25rem" iconName={IconName.Usdc} />,
+  },
+  deposit: {
+    title: 'Confirm deposit',
```

Comment:
I have a feeling these titles are already localized. change to their stringKeys and add stringGetter to `<DepositSteps>`?

Reply:
can i do this as part of 1 big localization push 😅